README.md

```diff
@@ -85,8 +79,8 @@ services as well as starting up worker threads.
 
 #### Building and Running separate CMR Applications
 
-This will build all of the applications, but will put each jar into the
-appropriate /target directory for each application. The command shown in step
+This will build all of the applications but will put each jar into the
+appropriate /target directory for each application. The command is shown in step
```

Comment:
The addition of `is` here is incorrect (see the rest of the sentence ...)

Reply:
dear @oubiwann i deleted my fork of the repo, because i don't want to keep a lot of repos without necessity. please feel free to merge this, and take 30 seconds to add the is to the docs. cheers. and thank you so much for being able to contribute to nasa. kind regards. T.

Reply:
Oh, dear ... deletion of branches before merge makes things difficult. I'll see what I can do.

Reply:
ok @oubiwann thank you your understanding. next time i follow this procedure, i will make sure to do it after merging. thank you for your understanding